### ISSUE TITLE
[Suggestion] /cmi point (as a console command)

### ISSUE THREAD

**Issue by mrfloris** [Zrips/CMI#1322]
Hey there, thank you for considering this suggestion.

Would it perhaps be possible to get support for /cmi point to run from console? This would allow us to make some magic wands or wizard wands that allow us to let players get a stick with attach command that they can use to point at others (a limited amount of times) - this is rather harmless but allows for some fun cosmetic effects.

!cc!cmi point [playerName] (type);;msg! [playername] WHOOoOSSh!

**Comment by Zrips** [comment]
Well there is one command currently disabled which could allow you to make unlimited amount of particles which could be used as cosmetic thing and to shoot particles from wand and similar stuff. Its disabled as its not 100% finished and its quite... Complex to use as its one liner thing, like in example warmup glyphead is circle;effect:flying_glyph;dur:5;pitchc:15;part:10;offset:0,1.7,0;radius:0.5;yawc:12;color:rs;pitch:90 variable and this one is one of more simpler ones. In general all those new particle effect which was added lately (holo click, rankup, warmup) are from that same command. 
So long story short, point command is not meant to be used in that way, we do have another one which could be utilized for 1000x better results but its not in release stage, for now.

**Comment by mrfloris** [comment]
> Well there is one command currently disabled which could allow you to make unlimited amount of particles which could be used as cosmetic thing and to shoot particles from wand and similar stuff. Its disabled as its not 100% finished and its quite... Complex to use as its one liner thing, like in example warmup glyphead is circle;effect:flying_glyph;dur:5;pitchc:15;part:10;offset:0,1.7,0;radius:0.5;yawc:12;color:rs;pitch:90 variable and this one is one of more simpler ones. In general all those new particle effect which was added lately (holo click, rankup, warmup) are from that same command.
> So long story short, point command is not meant to be used in that way, we do have another one which could be utilized for 1000x better results but its not in release stage, for now.


: mind blown : That's awesome! We can wait <3

This sounds fantastic and good things come to those who wait, thanks for the headsup.
